packages/ckeditor5-code-block/src/indentcodeblockcommand.js

```diff
@@ -76,7 +76,9 @@ export default class IndentCodeBlockCommand extends Command {
 			//		</codeBlock>
 			//
 			for ( const position of positions ) {
-				writer.insertText( this._indentSequence, position );
+				const indentSequenceTextElement = writer.createText( this._indentSequence );
+
+				editor.model.insertContent( indentSequenceTextElement, position );
```

Comment:
Maybe you could add some code comment explaining why this change is needed? Just to make sure that we will understand it in the future.